Flappy Bird Clone/Assets/Scripts/ScriptableObjects/PlayerInfo.cs

```diff
@@ -0,0 +1,16 @@
+using UnityEngine;
+
+namespace ScriptableObjects
+{
+    [CreateAssetMenu(fileName = "New Player Info", menuName = "Player Info")]
+    public class PlayerInfo : ScriptableObject
+    {
+        public float jumpForce = 5.0f;
```

Comment:
Out of curiosity: Why public has the same naming standard as privated serialized?

```cs
[SerializeField] private PlayerInfo playerInfo;
public float jumpForce = 5.0f;
```

Reply:
I thought to use camelCase on variables and attributes, and PascalCase on Classes and Structs.